lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +77,47 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            config = json.load(file)
+
+        return ConfigDTO(
+            seed_urls=config["seed_urls"],
+            total_articles_to_find_and_parse=config["total_articles_to_find_and_parse"],
+            headers=config["headers"],
+            encoding=config["encoding"],
+            timeout=config["timeout"],
+            should_verify_certificate=config["should_verify_certificate"],
+            headless_mode=config["headless_mode"])
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters are not corrupt.
         """
+        if not isinstance(self.config_dto.seed_urls, list):
+            raise IncorrectSeedURLError
+
+        for seed_url in self.config_dto.seed_urls:
+            if not re.match(r"https?://(www.)?", seed_url):
```

Comment:
we do not work with any website, right? we need seed urls to be exactly of the website we target

Reply:
fixed it